src/assets/stylesheets/screens/_blank-survey.scss

```diff
@@ -0,0 +1,29 @@
+.blank-survey {
+  margin-top: rem(50px);
+  
+  text-align: center;
+
+  @include media-breakpoint-up(sm) {
+    margin-top: rem(147px);
+  }
+
+  &__emo {
+    width: rem(48px);
+    height: rem(48px);
+    margin-bottom: rem(32px);
+
+    @include media-breakpoint-up(sm) {
+      width: rem(64px);
+      height: rem(64px);
+    }
+  }
+
+  &__message {
+    font-size: rem(18px);
```

Comment:
I'm not sure about to set the `font-size` as a variable because there is a lot of different font size in the Figma design.
Right now I have
- 13px
- 15px
- 17px
- 18px
- 22px
- 28px
- 34px

And still not sure if we will have more than this 🤔.
What do you think?

Reply:
These could be -

```
13px - xsm
15px - sm
17px - default
18px - md
22px - l
28px - xl
34px - xxl
```

We can always add an extra `x` if needed 🤣 . This convention is followed across the projects at nimble. For reference - https://github.com/nimblehq/compass/blob/development/_sass/_variables.scss#L74 @gutakk

Reply:
Fixed in 42295b078454de4c033562c6564e8ba65319bf89